serverless/knative_eventing/serverless-kn-source.adoc

```diff
@@ -9,7 +9,11 @@ toc::[]
  You can use `kn` to list and manage available event sources or event source types for use with Knative Eventing.
```

Comment:
For some reason, sentence and the following are building in code blocks. It's not immediately obvious to my why though.

Reply:
It looks OK to me in the preview? https://apiserver--ocpdocs.netlify.com/openshift-enterprise/latest/serverless/knative_eventing/serverless-apiserversource.html

Reply:
Figured it out, for some reason there was a slight one space indent on some of the lines :woman_shrugging: fixing it now, thank you!